src/rules_and_declarations.rs

```diff
@@ -157,6 +164,9 @@ pub trait QualifiedRuleParser {
     /// The finished representation of a qualified rule.
     type QualifiedRule;
 
+    ///
```

Comment:
This one seems to still be missing. A copy of the other `Error` docs is probably ok.